web/atlas.json

```diff
@@ -3829,7 +3829,7 @@
 {"id": "tx60mc", "submitted_by": "ZalcoMuse", "name": "Destinylocked", "description": "A crossover webcomic between Kingdom Hearts and Homestuck. The blue crown is the emblem of the main character, Sora.", "website": "https://mspfa.com/?s=12846&p=1", "subreddit": "/r/homestuck", "center": [696.5, 298.5], "path": [[692.5, 295.5], [700.5, 295.5], [700.5, 300.5], [692.5, 300.5]]},
 {"id": "tx60ep", "submitted_by": "lunatonesyt", "name": "CB", "description": "The logo for the GTA Family CleanBois", "website": "https://nopixel.fandom.com/wiki/Cleanbois", "subreddit": "", "center": [1353.5, 693.5], "path": [[1337.5, 677.5], [1336.5, 709.5], [1371.5, 709.5], [1370.5, 678.5]]},
 {"id": "tx60bn", "submitted_by": "sammyisrandom", "name": "Cleffa", "description": "Cleffa (Japanese: \u30d4\u30a3 Py) is a Fairy-type baby Pok\u00e9mon introduced in Generation II. Prior to Generation VI, it was a Normal-type Pok\u00e9mon. It is the baby evolution of Clefairy.", "website": "https://bulbapedia.bulbagarden.net/wiki/Cleffa_(Pok%C3%A9mon)", "subreddit": "/r/pokemon", "center": [1556.5, 1540.5], "path": [[1551.5, 1548.5], [1558.5, 1549.5], [1565.5, 1544.5], [1565.5, 1535.5], [1554.5, 1531.5], [1546.5, 1534.5], [1548.5, 1538.5], [1548.5, 1543.5]]},
-{"id": "tx5zl9", "submitted_by": "GreatHeroJ", "name": "Griefer Face (Unturned)", "description": "The 'griefer' face is a face that is usable by players in Unturned. It is Unturned's parody of the incredibly popular Trollface meme.", "website": "https://store.steampowered.com/app/304930/Unturned/", "subreddit": "/r/unturned", "center": [670.5, 566.5], "path": [[661.5, 557.5], [678.5, 557.5], [678.5, 574.5], [661.5, 574.5]]},
+{"id": "tx5zl9", "submitted_by": "GreatHeroJ", "name": "Griefer Face (Unturned)", "description": "The 'griefer' face is a face usable by players in Unturned. It is Unturned's parody of the incredibly popular Trollface meme.", "website": "", "subreddit": "", "center": [670.5, 566.5], "path": [[661.5, 557.5], [678.5, 557.5], [678.5, 574.5], [661.5, 574.5]]},

```

Comment:
Is this supposed to remove the website and subreddit?

Reply:
All three you mentioned were from this post, which just noticed they got back to me on https://www.reddit.com/r/placeAtlas2/comments/txt2um/entry_edits_unturned_zombie_face_griefer_face/